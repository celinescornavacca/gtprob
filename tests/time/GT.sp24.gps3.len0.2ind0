((((K0,(K1,K2)),(L0,(L1,L2))),(G0,(G1,G2))),(((((H1,H2),H0),((I1,I2),I0)),(((P0,(P1,P2)),(Q0,(Q1,Q2))),(((B0,(B1,B2)),(C0,(C1,C2))),(((T0,(T1,T2)),((U0,(U1,U2)),(V0,(V1,V2)))),(((E1,E2),E0),((F1,F2),F0)))))),((((W0,(W1,W2)),((X1,X2),X0)),((M0,(M1,M2)),(N0,(N1,N2)))),((A0,(A1,A2)),(((((O1,O2),O0),((R0,(R1,R2)),(S0,(S1,S2)))),((J1,J2),J0)),(D0,(D1,D2)))))));
