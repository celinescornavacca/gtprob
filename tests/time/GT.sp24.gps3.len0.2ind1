(((((C1,C2),C0),((D1,D2),D0)),(((G0,(G1,G2)),(((M1,M2),M0),(N0,(N1,N2)))),((((L1,L2),L0),((Q0,(Q1,Q2)),((R1,R2),R0))),(((H1,H2),H0),(I0,(I1,I2)))))),((((A1,A2),A0),((B0,(B1,B2)),((E0,(E1,E2)),(F0,(F1,F2))))),(((((W1,W2),W0),(X0,(X1,X2))),(K0,(K1,K2))),((((U0,(U1,U2)),((V1,V2),V0)),(((J1,J2),J0),(((O1,O2),O0),((P1,P2),P0)))),(((S1,S2),S0),(T0,(T1,T2)))))));
