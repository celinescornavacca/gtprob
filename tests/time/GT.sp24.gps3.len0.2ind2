((((((O1,O2),O0),((P1,P2),P0)),(((W1,W2),W0),((X1,X2),X0))),((D0,(D1,D2)),(((K0,(K1,K2)),(L0,(L1,L2))),((E1,E2),E0)))),((((I0,(I1,I2)),(J0,(J1,J2))),(((G0,(G1,G2)),((H1,H2),H0)),(((A1,A2),A0),(((((S0,(S1,S2)),(((U0,(U1,U2)),(V0,(V1,V2))),(T0,(T1,T2)))),(F0,(F1,F2))),((C1,C2),C0)),((B1,B2),B0))))),(((Q0,(Q1,Q2)),(R0,(R1,R2))),(((M1,M2),M0),(N0,(N1,N2))))));
