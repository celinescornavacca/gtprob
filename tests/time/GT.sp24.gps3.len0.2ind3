(((((((E1,E2),E0),((F1,F2),F0)),((B1,B2),B0)),((((L1,L2),L0),(M0,(M1,M2))),((J0,(J1,J2)),(K0,(K1,K2))))),(((C1,C2),C0),(((D1,D2),D0),((H0,(H1,H2)),((((((P1,P2),P0),((((U1,U2),U0),(V0,(V1,V2))),(Q0,(Q1,Q2)))),((I1,I2),I0)),((N0,(N1,N2)),((O1,O2),O0))),(((S1,S2),S0),(T0,(T1,T2)))))))),(((A1,A2),A0),((G0,(G1,G2)),(((R1,R2),R0),((W0,(W1,W2)),((X1,X2),X0))))));
