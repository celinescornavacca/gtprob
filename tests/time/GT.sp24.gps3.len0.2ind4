((((A1,A2),A0),((((E1,E2),E0),((L0,(L1,L2)),((S0,(S1,S2)),((T1,T2),T0)))),(((((U1,U2),U0),((V1,V2),V0)),(((H1,H2),H0),(((W1,W2),W0),(X0,(X1,X2))))),((((C1,C2),C0),(D0,(D1,D2))),((B1,B2),B0))))),(((((G1,G2),G0),(((Q1,Q2),Q0),((R1,R2),R0))),((((I0,(I1,I2)),((J0,(J1,J2)),((K1,K2),K0))),((M0,(M1,M2)),(N0,(N1,N2)))),((F1,F2),F0))),((O0,(O1,O2)),(P0,(P1,P2)))));
