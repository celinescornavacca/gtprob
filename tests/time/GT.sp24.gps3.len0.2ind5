(((((((Q1,Q2),Q0),((W0,(W1,W2)),((X1,X2),X0))),((G1,G2),G0)),(((D1,D2),D0),((((O0,(O1,O2)),(P0,(P1,P2))),(M0,(M1,M2))),(((N1,N2),N0),((((U1,U2),U0),(V0,(V1,V2))),((T1,T2),T0)))))),(((A1,A2),A0),((((J0,(J1,J2)),((((R0,(R1,R2)),((S1,S2),S0)),(L0,(L1,L2))),((K1,K2),K0))),((C1,C2),C0)),((B1,B2),B0)))),((((F1,F2),F0),(((H1,H2),H0),(I0,(I1,I2)))),((E1,E2),E0)));
