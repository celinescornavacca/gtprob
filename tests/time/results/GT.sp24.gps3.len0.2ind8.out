-108.139665684
